data-collection/test/test_from_scratch.py

```diff
@@ -175,10 +175,17 @@ def test_license_manager_grants(athena):
     data = athena_query(athena=athena, sql_query='SELECT * FROM "optimization_data"."license_manager_grants" LIMIT 10;')
     assert len(data) > 0, 'license_manager_grants is empty'
 
-def test_license_manager_licenses(athena):
```

Comment:
please can you check this? i do not think this is intentional

Reply:
Fixed now. Thanks for the catch. Yes it got overwritten during earlier commit